packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/cli/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,17 @@
+{
+  "pci_ai_dashboard_cli": "Command Line Interface (CLI)",
+  "pci_ai_dashboard_cli_description": "Utilisez notre interface ovhai afin d'accéder et gérer vos notebooks, jobs et apps en lignes de commande (CLI).",
+  "pci_ai_dashboard_cli_install": "Installation",
+  "pci_ai_dashboard_cli_install_restart_terminal": "Veillez à bien redémarrer votre terminal afin de pouvoir utiliser ovhai.",
+  "pci_ai_dashboard_cli_documentation_title": "Documentation",
```

Comment:
**nitpick:** This can be maintained as a constant.

Reply:
Same here, would not work for Spanish